src/common/enums/CMakeLists.txt

```diff
@@ -1,3 +1,7 @@
+if(${DEFAULT_FWD_REL_STORAGE})
```

Comment:
Why not keep this in the root cmake file together with other compile definitions?